project/src/const.js

```diff
@@ -0,0 +1,6 @@
+export const AppRoute = {
+  ROOT: '/',
+  LOGIN: '/login',
+  FAVORITES: '/favorites',
+  ROOM: '/offer',
```

Comment:
Предлагаю указать тут полноценный путь и генерировать из него url через `generatePath` из `react-router-dom`.
```suggestion
  ROOM: '/offer/:id',
```